pkg/providers/aws/provider_redis.go

```diff
@@ -387,6 +443,12 @@ func (p *RedisProvider) deleteElasticacheCluster(ctx context.Context, cacheSvc e
 
 	// check if replication group does not exist and delete finalizer
 	if foundCache == nil {
+		//TODO Manual testing only - should be removed
+		//will be handled properly in https://issues.redhat.com/browse/INTLY-8163
+		if err = networkManager.DeleteNetwork(ctx); err != nil {
+			msg := "failed to delete aws networking"
+			return croType.StatusMessage(msg), errorUtil.Wrap(err, msg)
+		}
```

Comment:
Are we saying this shouldn't be merged because of the TODO?

Reply:
same here, updated the comment inline and linked the jira, the work in the resource providers (redis and postgres) are essentially stubs, to allow for easy verification and testing during development. Proper provider specific implementation and testing will be handled in https://issues.redhat.com/browse/INTLY-8103